### DISCUSSION TITLE
Safe-Space for General Discussions!

### DISCUSSION THREAD

**Discussion by CanPixel** [CanPixel/BadOptics#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fengsemul** [comment]
Why is Avaritionism/Ego-Darwinism more left and statist than Ego-Capitalism and Anarcho-Capitalism? Disconnect between Description and Compass-Position? And in general how are Ideology-Balls located in gamified compass?

Assuming that Equality-Market and Authority-Liberty definition are the same as in politicalcompass.org and Polcompball-Wiki.

"An off-the-compass libertarian right-wing ideology..." - IdeologyDex

Maybe it's like this because "Social-Darwinism-Ball" is positioned less right-wing than the value "Markets" itself, making Anarcho-Capitalism more market-friendly than Ego-Darwinism in contra to popular social beliefs that Ego-Darwinism, Social Darwinism and Communalism is even edging Off-Compass compare to "IngSoc", "Egoism" being closer to their values in gamified compass.

**Comment by Fengsemul** [comment]
My idea is to add a Request-Tab after pressing "Reset Save File" due risk of misclicking it.
